package.json

```diff
@@ -30,7 +30,7 @@
 		"create-extension": "node ./tools/create-extension/index.js",
 		"cherry-pick": "node ./tools/cherry-pick/bin/run",
 		"sync-dependencies": "pnpm exec syncpack -- fix-mismatches",
-		"utils": "./tools/monorepo-utils/bin/run"
+		"utils": "node ./tools/monorepo-utils/dist/index.js"
```

Comment:
We don't need bin/run, its better to run the transpiled file where we can have full control over running the program and it will always be up to date.

Reply:
You would need to remove the `bin` entry to this file too. The idea here was to make room for this utility to become a script you could run from root, ie ` utils slack`. Although the name would be more better like `woocommerce ...`. Of course you'd need to `npm install -g .` to get it working.

Maybe that's all premature engineering though?

Reply:
@psealock oh ok, now I understand this a bit better you want to support installing the monorepo as a bin command that can be run from "anywhere" not project root.

Can we just do that when we decide to actually publish it? We still don't need to have a separate bin/run script for that, we can just during the build before publish, also output the JS into a `bin` folder and have a single script like `node ./path/to/index.js`

Reply:
>Can we just do that when we decide to actually publish it?

Sure that's fine.

>We still don't need to have a separate bin/run script for that, we can just during the build before publish, also output the JS into a bin folder and have a single script like node ./path/to/index.js

This was my original approach as well. It turns out permissions is an issue as TS compiled can't be run directly like that. See https://github.com/microsoft/TypeScript/issues/16667. Creating the `bin/run` script was the work around.

Reply:
@psealock even so, we can just run a script to adjust the permissions after build. so it would be like 

`
tsc
chmod +x the-main-file.js
`

Reply:
That was my second solution! I didn't like that either, but I suppose now this is an issue for a later date